examples-gallery/advanced/plot_park2004.py

```diff
@@ -54,6 +55,25 @@
 eom = h.first_order_implicit()
 MathJaxRepr(sm.simplify(eom))
 
+# %% Generate Stability Constraints
+# ---------------------------------
+#
+A, B, C, D = h.closed_loop_state_space()
+d1, d2, d3, d4, d5, d6, d7, d8 = sm.symbols('d1, d2, d3, d4, d5, d6, d7, d8',
+                                            real=True)
+A_ = sm.Matrix([[0, 0, 1, 0],
+                [0, 0, 0, 1],
+                [d1, d2, d3, d4],
+                [d5, d6, d7, d8]])
+B_ = sm.ones(4, 2)
+C_ = sm.eye(4)
+D_ = sm.zeros(*B_.shape)
+ss = ctrl.StateSpace(A_, B_, C_, D_)
+ineq = ss.get_asymptotic_stability_conditions()
```

Comment:
This fails with the matrices from `h.closed_loop_state_space()`. But I can do this substitution to get it to work.